Report.py

```diff
@@ -0,0 +1,58 @@
+from Computations import Computations
+import calendar
+
+
+class Report:
+
+    @staticmethod
+    def find_maximum_temperature_in_year(year, records):
+        month, day, temperature = Computations.find_maximum_temperature_in_year(year, records)
+        print("Highest: " + str(temperature) + "C on " + calendar.month_name[month] + " " + str(day))
+
+    @staticmethod
+    def find_minimum_temperature_in_year(year, records):
```

Comment:
Same name in `Report` and `Computations`? Doesn't it confuses? Kindly look into appropriate naming.

Reply:
Same thing for all other methods as well.